synapse/rest/client/login.py

```diff
@@ -536,7 +536,7 @@ def _get_auth_flow_dict_for_idp(idp: SsoIdentityProvider) -> JsonDict:
 
 
 class RefreshTokenServlet(RestServlet):
-    PATTERNS = (re.compile("^/_matrix/client/v1/refresh$"),)
+    PATTERNS = client_patterns("/refresh$")
```

Comment:
Was this valid for `r0`? Should this be:

```suggestion
    PATTERNS = client_patterns("/refresh$", releases=("v3",), unstable=False)
```

Reply:
It hasn't been in the spec as r0, but implementing it as r0 was technically allowed (implementing stable prefixes is allowed immediately after a MSC is merged, and the MSC was merged before everything was bumped to v3).

Reply:
Regardless, I think including it as unstable is not wanted. We've been trying to reduce the number of unstable endpoints we expose.